openshift/scripts/oc_cleanup.sh

```diff
@@ -35,6 +35,7 @@ OC_CLEAN_FORECAST_CRONJOB="oc -n ${PROJ_DEV} ${DELETE_OR_GET} cronjob/bcfw-p1-fo
 OC_CLEAN_EC_GDPS_CRONJOB="oc -n ${PROJ_DEV} ${DELETE_OR_GET} cronjob/env-canada-gdps-${NAME_APP}-${SUFFIX}"
 OC_CLEAN_EC_HRDPS_CRONJOB="oc -n ${PROJ_DEV} ${DELETE_OR_GET} cronjob/env-canada-hrdps-${NAME_APP}-${SUFFIX}"
 OC_CLEAN_EC_RDPS_CRONJOB="oc -n ${PROJ_DEV} ${DELETE_OR_GET} cronjob/env-canada-rdps-${NAME_APP}-${SUFFIX}"
+OC_DELETE_EC_PODS="oc -n ${PROJ_DEV} get pods -o name | grep -E 'env-canada-(gdps|rdps|hrdps)-${NAME_APP}-${SUFFIX}' | xargs oc ${DELETE_OR_GET}"
```

Comment:
```suggestion
OC_DELETE_EC_PODS="oc -n ${PROJ_DEV} ${DELETE_OR_GET} pods -o name | grep -E 'env-canada-(gdps|rdps|hrdps)-${NAME_APP}-${SUFFIX}' | xargs oc ${DELETE_OR_GET}"
```

Reply:
Nope. The first command, will always be an oc get.
Step #1 - get a list of pods
Step #2 - filter down to only giving env canada pods for this pr
Step #3 - delete each pod (this is the DELETE_OR_GET)

Reply:
Ah I can see `${DELETE_OR_GET}` at the end now